### PR TITLE
TELCODOCS-1480 - PTP T-GM dual NIC config for D-RAN use cases

### PR DESCRIPTION
**Pull request by aireilly**
Adds docs for [TELCODOCS-1480](https://issues.redhat.com//browse/TELCODOCS-1480) - PTP T-GM dual NIC config for D-RAN use cases.

Version(s):
enterprise-4.15

Issue:
https://issues.redhat.com/browse/TELCODOCS-1480

Link to docs preview:
* https://69941--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/configuring-ptp#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp
* https://69941--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/about-ptp#ptp-dual-nics_about-ptp

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by jzding** [comment]
for ptpconfig example you can add a few highlights below (line numbers from my [PR](https://github.com/openshift-kni/cnf-features-deploy/pull/1756/files#diff-fa7d288cd5725663858530f03e8397f0ee37d737bb06b2454f24ff418eddb1a8)):
for line 29-38 about pin configurations. In this example 2 NICs are connected via SMA1 connections, where pin "2 1" enables 1PPS OUT, and pin "1 1" enables 1PPS IN.
line 106-113: ts2phc parameters can be set for each NIC
line 112-113: add value to compensate for the delay from SMA cable. The value should be a result of measurement from lab testing.
line 213: boundary_clock_jbod need to set to 1 to enable support of multiple cards.

**Review by jzding — APPROVED** [review]
/lgtm

File: modules/nw-ptp-dual-wpc-hardware-config-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-ptp-dual-wpc-hardware-config-reference_{context}"]
+= Intel Westport Channel E810 dual NIC hardware configuration reference
```
Comment (skrthomas) [inline modules/nw-ptp-dual-wpc-hardware-config-reference.adoc:7]: Minor noticing, but I note that another topic is titled "Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs" and I wonder if the title here should swap around the E810 and dual. #notmyprodarea so I defer to you. Just a minor thing I noticed.

File: modules/nw-ptp-dual-wpc-hardware-config-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+|`spec.profile.ts2phcConf`
+|Use the `ts2phcConf` field to configure parameters for NIC one and NIC two.
+Configure the `ts2phc.extts_correction` value for NIC two to compensate for the delay that is incurred for the specific SMA cable and cable length that you use.
+The value that you configure depends on your specific measurements and SMA1 cable length.
+
+|`spec.profile.ptp4lConf`
+|Set the value of `boundary_clock_jbod` to 1 to enable support for multiple NICs.
+|====
+
```
Comment (skrthomas) [inline modules/nw-ptp-dual-wpc-hardware-config-reference.adoc:37]: remove this extra line

**Review by skrthomas — COMMENTED** [review]
Looks great! Just a couple of minor comments.

**Review by jzding — APPROVED** [review]
/lgtm

**Comment by aireilly** [comment]
@jzding is this ready for @Bonnie-Block to review?

**Review by jzding — APPROVED** [review]
/lgtm

**Comment by jzding** [comment]
There are 2 more changes:
1. Change help text for the openshift_ptp_clock_class metrics to  "6 = Locked, 7 = PRC unlocked in-spec, 52/187 = PRC unlocked out-of-spec, 135 = T-BC holdover in-spec, 165 = T-BC holdover out-of-spec, 248 = Default, 255 = Slave Only Clock".
Introduced by https://github.com/openshift/linuxptp-daemon/pull/266

2. Naming changes for dual card config:
Changed i/f names `iface_master` to `iface_nic1` and `iface_master_1` to `iface_nic2` to avoid confusion.
Introduced by https://github.com/openshift-kni/cnf-features-deploy/pull/1807

**Comment by aireilly** [comment]
@jzding PTAL for OCPBUGS-28651 updates and https://github.com/openshift/openshift-docs/pull/70235 for the clock class metrics

**Comment by aireilly** [comment]
NOTE: This is not merged to 4.15 yet - waiting for the feature to be released on 4.15.

**Pull request merged**